site-src/geps/gep-1426.md

```diff
@@ -114,37 +113,44 @@ Services supported as `backendRefs` SHOULD be consistent with expectations for N
-* NOT already specified as a `backendRef` of any _other_ `HTTPRoute` with a `Service` `parentRef`
+* NOT already specified as a `backendRef` of any _other_ `Route` with a `Service` `parentRef`
    * This restriction is intended to prevent route "inclusion/delegation" patterns or circular references from multiple tiers of routing rules _within mesh configuration_.
-   * A `Service` MAY still be listed as a `backendRef` on a different `HTTPRoute` which only targets a `Gateway` as a `parentRef`. The specific behavior of how Gateways might interact with mesh routing rules will be defined in a future GEP.
-    * A `Service` MAY still be listed as a `backendRef` on the _same_ `HTTPRoute` where it is specified as a `parentRef`, e.g. to only split off some subset of traffic from the default backend endpoints.
+   * A `Service` MAY still be listed as a `backendRef` on a different `Route` which only targets a `Gateway` as a `parentRef`. The specific behavior of how Gateways might interact with mesh routing rules will be defined in a future GEP.
+    * A `Service` MAY still be listed as a `backendRef` on the _same_ `Route` where it is specified as a `parentRef`, e.g. to only split off some subset of traffic from the default backend endpoints.
     * This could be computationally expensive to validate.
 
 This should be enforced by controller validation, which should set an `Accepted: { status: False }` condition with a corresponding new `ParentRefConflict` `RouteConditionReason`.
 
-The logic described here is a bit verbose, but could be validated with conformance tests, and is intended to allow an `HTTPRoute` configuring mesh traffic rules to additionally specify a `Gateway` `parentRef` (or `HTTPRoute` `parentRef` if [GEP-1058: Route Inclusion and Delegation](https://github.com/kubernetes-sigs/gateway-api/pull/1085) moves forward) to apply the same routing rules when exposing a mesh service outside the cluster.
+The logic described here is a bit verbose, but could be validated with conformance tests, and is intended to allow an `Route` configuring mesh traffic rules to additionally specify a `Gateway` `parentRef` (or `Route` `parentRef` if [GEP-1058: Route Inclusion and Delegation](https://github.com/kubernetes-sigs/gateway-api/pull/1085) moves forward) to apply the same routing rules when exposing a mesh service outside the cluster.
+
+### Route types
+
+All types currently defined in the gateway-api core (`HTTP`, `GRPC`, `TCP`, `TLS`, and `UDP`) are available for use in a Mesh implementation.
+
+If multiple routes with of different type both bind to the same Service and Port pair, the same resolution rules as defined in [`listener.allowedRoutes`](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1beta1.AllowedRoutes) apply.
```

Comment:
This is actually ambiguous in Gateway as far as I can tell. I asked for clarity and will update either the `allowedRoutes`, this sentence, or both once I understand the desired intent.